### PR TITLE
XTS and RTS images

### PR DESCRIPTION
**Pull request by gytis**

### CODE REVIEW

**Comment by gytis** [comment]
Could somebody take a look at this, please?

File: rts-wildfly/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Use latest jboss/wildfly image as the base
+FROM jboss/wildfly
```
Comment (goldmann) [inline rts-wildfly/Dockerfile]: Please add a tag here: `:8.2.0.Final` for example.

File: rts-wildfly/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+# Make standalone-rts.xml configuration default
+RUN cp /opt/jboss/wildfly/docs/examples/configs/standalone-rts.xml /opt/jboss/wildfly/standalone/configuration/standalone.xml
```
Comment (goldmann) [inline rts-wildfly/Dockerfile]: You can use `$JBOSS_HOME` env variable everywhere.

File: rts-wildfly/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+# Set the default command to run on boot
+# This will boot WildFly in the standalone rts mode and bind to all interfaces
+CMD ["/opt/jboss/wildfly/bin/standalone.sh", "-b", "0.0.0.0"]
```
Comment (goldmann) [inline rts-wildfly/Dockerfile]: You do not need to specify the `CMD` again, it'll be reused from the `jboss/wildfly` image.

**Comment by gytis** [comment]
@goldmann, I've updated commits

**Comment by goldmann** [comment]
It would be probably good to follow the convention to name the readme file as `README.md`. I'm always looking for files in upper case when I look for readme's and licenses.

Otherwise looks good!

**Comment by gytis** [comment]
@goldmann, sorry for the late rely, I was away.
Anyway, I have now updated the readme file names. (I didn't know that it's the convention to have them capitalised).

**Comment by goldmann** [comment]
Thanks!

**Pull request merged**